### PR TITLE
stormssh (0.6.5) new formula

### PR DESCRIPTION
**Pull request by youtux**
I'm willing to add stormssh, a CLI/web-based interface to manage the `.ssh/config` file. It can be installed using pip, but it has a lot of dependencies and it would be nice to let brew package it so that the system python stays clean.

### CODE REVIEW

File: Library/Formula/stormssh.rb [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    system bin/"storm", "--version"
```
Comment (mikemcquaid) [inline Library/Formula/stormssh.rb]: Can remove this.

File: Library/Formula/stormssh.rb [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+    system bin/"storm", "add", "--config", "sshconfig", "aliastest", "user@example.com:22"
+
+    expected_output = <<-EOS.undent
+      Host aliastest
+          hostname example.com
+          user user
+          port 22
+    EOS
+    assert_equal expected_output, sshconfig_path.read
```
Comment (mikemcquaid) [inline Library/Formula/stormssh.rb]: :+1:

File: Library/Formula/stormssh.rb [hunk 3]
```diff
@@ -0,0 +1,88 @@
+class Stormssh < Formula
+  homepage "https://github.com/emre/storm"
+  url "https://pypi.python.org/packages/source/s/stormssh/stormssh-0.6.5.tar.gz"
+  sha1 "3788fca510dc1a46aa21adea98749c354d85b3cc"
+  head "https://github.com/emre/storm.git"
+
+  conflicts_with "storm"
```
Comment (mikemcquaid) [inline Library/Formula/stormssh.rb]: Needs a reason.

**Comment by emre** [comment]
thanks @youtux :+1:

**Pull request closed without merging**